docs/kh3/crimovie.md

```
@@ -0,0 +1,159 @@
+| en / ja       | mv_secret.usm     | [Secret Movie](https://www.youtube.com/watch?v=U7YaNjUe01o)                                                          |
+
+### main
+
+Files ending with with ```cXX``` are usually flashback scenes or overlays within a bigger cutscene.
+
+Base Path: ```<Installation Path>\KINGDOM HEARTS III\Content\CriMovie\main```
+
+| **Folder**     | **File Name**             | **Description**                                                                             | **World**                  |
+|----------------|---------------------------|---------------------------------------------------------------------------------------------|----------------------------|
+| Base Path      | ca406_shot010.usm         | [Air Raid](https://www.youtube.com/watch?v=LXotoQgGnVo)                                     | The Caribbean              |
+| Base Path      | ew_timelapse.usm          | [Timelapse of Clouds in Tutorial Section]()                                                 | The Final World            |
```

markdownlint-cli2 (0.18.1)

No empty links

(MD042, no-empty-links)